pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -180,7 +180,7 @@ type RevisionSpec struct {
 
 	// TimeoutSeconds holds the max duration the instance is allowed for responding to a request.
 	// +optional
-	TimeoutSeconds *metav1.Duration `json:"timeoutSeconds,omitempty"`
+	TimeoutSeconds int64 `json:"timeoutSeconds,omitempty"`
```

Comment:
Should you use the type `*int64` ? Then the json value `null` would still remain valid if any clients were sending that.

Reply:
Given that we haven't released what we had, I'm not super interested in compatibility.  They'll also be broken if they pass anything but nil, so...

Reply:
> Given that we haven't released what we had, I'm not super interested in compatibility. 

Sounds good.